packages/@okta/vuepress-site/docs/reference/releases-at-okta/index.md

```diff
@@ -1,71 +1,72 @@
 
-Changes, regardless of lifecycle stage, are always reported in the [Okta API Products Release Notes](/docs/release-notes/).
+Changes, regardless of life cycle stage, are always reported in the [Okta API Products Release Notes](/docs/release-notes/).
 
 ## Quick Reference Table
 
-| Description                | Beta (High touch)    | Beta (Low touch)    | EA                   | GA                   | Deprecated |
-| :------------              | :------------------: | :-----------------: | :---:                | :---:                | :---:      |
-| Contact with Product Team  | X                    |                     |                      |                      |            |
-| API  Changes               | Subject to change    | Subject to change   | Backwards compatible | Backwards compatible | N/A        |
-| Okta Support               |                      |                     | X                    | X                    |            |
-| Service-level agreements   |                      |                     | X                    | X                    | X          |
-| Announced in Release Notes |                      |                     | X                    | X                    | X          |
-| In preview orgs            | By invitation        | By invitation       | By request           | X                    | X          |
-| In production orgs         |                      |                     | By request           | X                    | X          |
-| Documentation              | Limited              | Limited             | X                    | X                    | X          |
+| Description                | Beta                 | EA                   | GA                   | Deprecated |
+| :------------              | :------------------: | :-----------------:  | :------------------: | :---------:|
+| Contact with Product Team  | &check;              | &cross;              | &cross;              | NA         |
+| API  Changes               | Subject to change    | Backwards compatible | Backwards compatible | N/A        |
+| Okta Support               | &cross;              | &check;              | &check;              | &check;    |
+| Service-level agreements   | &cross;              | &check;              | &check;              | &check;    |
+| Announced in Release Notes | &cross;              | &check;              | &check;              | &check;    |
+| In preview orgs            | By invitation or self-service, depending on the feature | By request or self-service, depending on the feature | &check;| &check;|
+| In production orgs         | &cross;              | By request or self-service, depending on the feature| &check;| &check; |
+| Documentation              | Limited              | &check;              | &check;              | NA         |
```

Comment:
:) Using the checkmarks too makes it much clearer.

Reply:
Jakub's idea - I agree - it helps a lot.